docs/source/tutorial/package_a_project.md

```diff
@@ -111,29 +111,22 @@ The resulting `.whl` packages only contain the Python source code of the Kedro p
 
 The project configuration is provided separately in a `tar.gz` file, also inside the `dist` folder. This compressed version of the config files excludes any files inside the `local` directory.
 
-### Package recipients
+### Run a packaged project
 
-Recipients of the `.whl` file need to have Python and `pip` on their machines, but do not need to have Kedro installed.
-
-A recipient can install the project by calling:
+To run a packaged project it must first be installed. To install the project, run the following command:
 
 ```bash
 pip install <path-to-wheel-file>
 ```
 
-An executable, `spaceflights`, is placed in the `bin` subfolder of the Python install folder, so the project can be run as follows:
+Once your project is installed, it  can be run in two ways:
 
-```bash
-python -m spaceflights
-```
+1. Using the project's CLI.
```

Comment:
Numbered bullets imply a sequence, that is, you have to do both. I would instead have an "either X or Y" statement as a sentence rather than a list.

Reply:
Thanks for explaining this, that makes sense.